src/lib/storage/snapshots_finish.rb

```diff
@@ -0,0 +1,48 @@
+require "yast2/fs_snapshot"
+require "installation/finish_client"
+
+module Storage
+  class SnapshotsFinish < ::Installation::FinishClient
+    include Yast::I18n
+
+    def initialize
+      textdomain "storage"
+
+      Yast.import "Mode"
+      Yast.import "StorageSnapper"
+      Yast.include self, "installation/misc.rb"
+    end
+
+    # Write configuration
+    #
+    # It creates a snapshot when no second stage is required and
+    # Snapper is configured.
+    #
+    # @return [TrueClass,FalseClass] True if snapshot was created;
+    #                                otherwise it returns false.
+    def write
+      if !second_stage_required? && StorageSnapper.configure_snapper?
```

Comment:
Should FsSnapshot.configure? do the trick? https://github.com/yast/yast-yast2/blob/master/library/system/src/lib/yast2/fs_snapshot.rb#L72